gammapy/maps/region/ndmap.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, geom, data=None, dtype="float32", meta=None, unit=""):
         self.meta = meta
         self._unit = u.Unit(unit)
 
-    def plot(self, ax=None, axis_name=None, **kwargs):
+    def plot(self, ax=None, axis_name=None, time_format="iso", **kwargs):
```

Comment:
Since our Maps are very general, I dont know how nice it is to have explicit arguments for configuring each axis type... Maybe having another `kwargs` would be better?

Reply:
So you propose something along the lines of:

```python3
map.plot(axis_name="time", axis_plot_kwargs={"time_format": "mjd"} )
``` 
Where `axis_plot_kwargs` would be forwarded to `axes[axis_name].format_plot_xaxis(**axis_plot_kwargs)`.

I tend to agree, except that right now we do not have arguments for e.g. `MapAxis.format_plot_xaxis()`, so right now there is no other uses case than the time format. If we can imagine to have other use cases then it is probably a good idea.

Reply:
@adonath : Yes, we just discussed this, and at present we dont see other use cases. So maybe this can remain as is now, and in future we can move it to kwargs. This should be backwards compatible

Reply:
@adonath : Yes, we just discussed this, and at present we dont see other use cases. So maybe this can remain as is now, and in future we can move it to kwargs. This should be backwards compatible